paddle/fluid/operators/split_selected_rows_op.h

```diff
@@ -72,10 +72,11 @@ class SplitSelectedRowsOpKernel : public framework::OpKernel<T> {
     for (size_t i = 0; i < outs_rows_idx.size(); ++i) {
       auto rows_idx = outs_rows_idx[i];
       outs[i]->set_height(height_sections[i]);
+      auto dims = x->GetCompleteDims();
+      dims[0] = rows_idx.size();
```

Comment:
Does it works well when `rows_ids.size()=0` ?

Reply:
Sure, some element of `outs_rows_idx` may be empty when distributed training, that means that we don't need to call `mutable_data` for that, and the tensor would also not beed initliazed.